eZ/Publish/API/Repository/Tests/Values/User/Limitation/LanguageLimitationTest.php

```diff
@@ -235,4 +249,136 @@ public function testPublishVersionIsNotAllowedIfModifiedOtherTranslations(array
         $this->expectException(UnauthorizedException::class);
         $contentService->publishVersion($folderDraft->getVersionInfo());
     }
+
+    public function testPublishVersionTranslation(): void
```

Comment:
Nitpick: I prefer having exceptions documented, so IDE doesn't report them as not handled:
```suggestion
    /**
     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
     */
    public function testPublishVersionTranslation(): void
```